src/workers.jl

```diff
@@ -120,7 +120,23 @@ function trigger_profile(w::Worker, timeout_profile_wait, from::Symbol=:manual)
         elseif Sys.isbsd()
             kill(w.process, 29)  # SIGINFO
         end
-        sleep(timeout_profile_wait) # Leave time for it to print the profile.
+        sleep(timeout_profile_wait) # Leave time to output the profile.
+    end
+    return nothing
+end
+
+# Spawn GDB to request thread backtraces followed by invoking jl_print_task_backtraces
+# to get task backtraces. We wait for GDB to finish so no sleep is needed. Thread backtraces
+# are written to `gdb.btall`, which is overwritten if present.
+function trigger_backtraces(w::Worker, from::Symbol=:manual)
+    if Sys.islinux()
+        @debug "using GDB to get thread and task backtraces on worker $(w.pid) from $from"
+        gdb_cmd = `gdb -ex "handle SIGSEGV noprint nostop pass" -ex "handle SIGUSR2 noprint nostop pass" -ex "set pagination 0" -ex "set logging overwrite on" -ex "set logging file gdb.btall" -ex "set logging redirect on" -ex "set logging enabled on" -ex "thread apply all bt" -ex "set logging enabled off" -ex "call jl_print_task_backtraces(1)" --batch -p $(w.pid)`
```

Comment:
there is a lot going on here... i think we need a comment (maybe a long one) 

perhaps we write this out over multiple lines so we can add comments about blocks of `-ex` commands, like

```suggestion
        gdb_cmd = Cmd([
        "gdb",
        # comment here about SIGSEGV and SIGUSR2
        "-ex", "handle SIGSEGV noprint nostop pass",
        "-ex", "handle SIGUSR2 noprint nostop pass",
        "-ex", "set pagination 0",
        "-ex", "set logging overwrite on",
        "-ex", "set logging file gdb.btall",
        "-ex", "set logging redirect on",
        # comment here about getting traces for all threads
        "-ex", "set logging enabled on",
        "-ex", "thread apply all bt",
        "-ex", "set logging enabled off",
        # comment here...
        "-ex", "call jl_print_task_backtraces(1)",
        "--batch", "-p", "$(w.pid)"
    ])
```

e.g. i'm thinking a block like this needs explaining (probably not to a `gdb` connoisseur but to some of us less familiar with this wizardry):
```
-ex "set logging enabled on" -ex "thread apply all bt" -ex "set logging enabled off"
```